src/src/AdminBundle/Controller/MailingListController.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace AdminBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use AdminBundle\Entity\Attendee;
+use AdminBundle\Entity\Event;
+use AdminBundle\Entity\EventType;
+use AdminBundle\Entity\Registration;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Doctrine\ORM\Query\Expr\Join;
+
+class MailingListController extends Controller
+{
+    /**
+     * @Route("/mailing_list/{page}", name="mailing_list", defaults={"page"=1})
+     * @Method("GET")
+     */
+    public function indexAction($page, Request $request)
+    {
+        $numItemsPerPage = 3;
+        $countPages = 0;
+        $mailingList = [];
+        $em = $this->getDoctrine()->getManager();
+        $defaultData = array('nameEmail' => '', 'event' => '', 'isSubscribed' => false);
+        $form = $this->createFormBuilder($defaultData)
```

Comment:
Vytiahni toto prosim do na to urceneho suboru.

Reply:
kde to mam vytiahnut?

Reply:
Forms.php pozri štruktúru projektu